### PR TITLE
Changed payload ticks to be more visible

### PR DESCRIPTION
**Pull request by annelausf**
Payload changes were displayed by red and black ticks that were not that visible and had no explanation on them. 
Fix: 
Changed the black ticks to white ticks to increase visibility.
Added a hover over so that there is information on what the ticks represent. 
Added a toggle to turn off payload change information. 

Before:
Screen Shot 2021-09-28 at 4 37 13 PM
 
After:
Screen Shot 2021-09-28 at 4 37 55 PM

### CODE REVIEW

**Comment by venkatramreddykunta** [comment]
**Offline Discussion**: I remember we agreed on moving "Show Payload Changes" Toggle button before Submit. 
Checking if there was any reason for moving it after Submit Button ?

**Comment by kritikapradhan** [comment]
Pls update bindata webfile as well

File: pkg/sloop/webserver/webfiles/sloop_ui.js [hunk 1]
```diff
@@ -333,10 +380,10 @@ function getHeatmapContent(d) {
         let severityCode = severity.get(splitText[1]);
         let severityColor = palette.severity[severityCode];
         return `<tr>
-                 <td> <b> ${splitText[0]} </b> </td>
-                 <td> <b> ${splitText[2]} </b> </td> 
-                 <td> <b style="color:${severityColor}">${severityText}</b> </td>
-                 </tr>` + r
+                <td> <b> ${splitText[0]} </b> </td>
```
Comment (mengyaoyang11) [inline pkg/sloop/webserver/webfiles/sloop_ui.js:383]: why we change indentation here?

Comment (annelausf) [reply pkg/sloop/webserver/webfiles/sloop_ui.js]: Fixed, thanks!

File: pkg/sloop/webserver/webfiles/index.html [hunk 2]
```diff
@@ -75,7 +75,15 @@
 
             <input type="submit">
         </form>
-
+        <!-- Toggle switch to show payload changes -->
+        <div class="switchdiv">
```
Comment (venkatramreddykunta) [inline pkg/sloop/webserver/webfiles/index.html:79]:
```suggestion
        <div class="payload-changes-switch-container">
```

Comment (venkatramreddykunta) [reply pkg/sloop/webserver/webfiles/index.html:79]: **Suggestion**: Add classes based on the need
**Example**: switchpara can be accessed `.payload-changes-switch-container > p`

**Comment by annelausf** [comment]
> **Offline Discussion**: I remember we agreed on moving "Show Payload Changes" Toggle button before Submit. Checking if there was any reason for moving it after Submit Button ?

@venkatramreddykunta After discussion with team, placing the toggle separately will allow users to use the function without having to submit the form each time and will help reduce the number of backend queries.

File: pkg/sloop/webserver/webfiles/index.html (hunk 2 above)
Comment (annelausf) [reply pkg/sloop/webserver/webfiles/index.html:79]: I edited it to `switch-container` and `.switch-container p`. If we create other toggles in the future - we can use the same class name.

**Comment by venkatramreddykunta** [comment]
> > **Offline Discussion**: I remember we agreed on moving "Show Payload Changes" Toggle button before Submit. Checking if there was any reason for moving it after Submit Button ?
> 
> @venkatramreddykunta After discussion with team, placing the toggle separately will allow users to use the function without having to submit the form each time and will help reduce the number of backend queries. In the future, we can change the design to remove separation between the query filter form and the ui changes toggle.

Sounds good !. 
Ya in the future we could **remove Submit button** and add **fire events for each of the filters**. 
**Example**: End Time, Look Back Range, Filter Namespace, Filter Kind, Sort, Updating Name Filter (We could have **Save/Update** filter button Besides Name Filter Text Field)

CC: @sana-jawad 

filters

**Review by venkatramreddykunta — APPROVED** [review]

File: pkg/sloop/webserver/webfiles/sloop_ui.js [hunk 3]
```diff
@@ -6,13 +6,13 @@
...
 
 const palette = {
-        baseDark: ["#2E3440", "#3B4252", "#434C5E", "#4C566A"],
-        baseLight: ["#D8DEE9", "#E5E9F0", "#ECEFF4", "#F3F3FE"],
-        primary: ["#8FBCBB", "#88C0D0", "#81A1C1", "#5E81AC"],
-        highlight: ["#BF616A", "#D08770", "#EBCB8B", "#A3BE8C",
-            "#B48EAD","#ADA8B6","#da7650","#D496A7","#ABC4AB",
-            "#A4A6D2","#91BEF2","#97B1A6","#8A9B68"],
-        severity: ['#4BD855', '#E0E000', '#D84B4B'],
+    baseDark: ["#2E3440", "#3B4252", "#434C5E", "#4C566A"],
```
Comment (sana-jawad) [inline pkg/sloop/webserver/webfiles/sloop_ui.js:9]: intentional indentation changes?

**Review by sana-jawad — APPROVED** [review]

File: pkg/sloop/webserver/webfiles/sloop_ui.js (hunk 3 above)
Comment (annelausf) [reply pkg/sloop/webserver/webfiles/sloop_ui.js:9]: Yes, other places in the code mostly used single tab so I changed it.

**Review by mengyaoyang11 — APPROVED** [review]
LGTM

**Pull request merged**